### ISSUE TITLE
Launch with last opened project

### ISSUE THREAD

**Issue by goblinJoel** [rpgtoolkit/editor#13]
I think it would be nice if launching the toolkit automatically opened the last opened project in the editor. I suppose this would have to be on a per-user basis (or just fail to do so if we don't have permissions for the last opened project).

In the case that there is no last opened project, or we don't have permissions for it, perhaps it should open the open project dialogue on launch. 

Similarly, it might be nice to have a recently opened files and/or projects menu, which we could discuss as another issue if you want.

Currently, I've hacked the immediate open project dialogue in as non-committed changes to speed testing (along with immediate open file dialogue after project is selected, but I don't see that as an end-user feature), but I think this might be a nice feature in general.

**Comment by chutchinson** [comment]
I would argue to implement standard MRU functionality and throw in a configuration option on what to do after the main window initializes (e.g. environment startup).

**Comment by goblinJoel** [comment]
Seems reasonable. I had to google MRU, but that is more or less what I'm asking for: "a recently opened files and/or projects menu" and a way to have it start by opening the most recent project or the open project window. I would assume this isn't hard to implement, but then I've seen programs with MRU menus that seem to forget all the time or don't remember the same stuff as Windows's right-click menu does. Presumably there's some catch.

**Comment by swordmaster2k** [comment]
Yeah I actually have RPGCode Express opening with the last project. It does it based on an editor configuration file, pretty straight forward stuff to implement. It was XML based, obviously it ran a bunch of checks to ensure that the project path still existed etc.
